### PR TITLE
Port AutoMMPredictor to Tabular + Add kaggle house competition example + Improve example readmes

### PR DESCRIPTION
**Pull request by sxjscience**
*Issue #, if available:*

*Description of changes:*
 

- Port AutoMMPredictor to Tabular, with the key "AG_AUTOMM".
- Improve the FT-Transformer example and other example readmes.
- Add a new example about Kaggle california house price prediction competition.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tabular/src/autogluon/tabular/models/automm/automm_model.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+from autogluon.core.models import AbstractModel
+
+logger = logging.getLogger(__name__)
+
+
+class AutoMMPredictorModel(AbstractModel):
+    _NN_MODEL_NAME = 'automm_model'
+
+    def __init__(self, **kwargs):
+        """Wrapper of autogluon.text.automm.AutoMMPredictor. #TODO Change it to autogluon.multimodal.AutoMMPredictor later.
```
Comment (zhiqiangdon) [inline tabular/src/autogluon/tabular/models/automm/automm_model.py:23]: Wrong module path: `autogluon.text.automm.AutoMMPredictor`

**Comment by szha** [comment]
Job PR-1872-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1872/1/index.html

**Comment by szha** [comment]
Job PR-1872-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1872/4/index.html

**Review by Innixma — APPROVED** [review]
LGTM!

**Pull request merged**